helm/odigos/templates/instrumentor/clusterrole.yaml

```diff
@@ -67,6 +67,12 @@ rules:
     verbs:
       - update
 {{- end }}
+  - apiGroups:
+      - operator.odigos.io
+    resources:
+      - odigos/finalizers
+    verbs:
+      - update
```

Comment:
if we install with helm, I guess we don't really need the operator permissions, right? will we even have the `operator.odigos.io` api group and `odigos` CRD in helm?

We can have this for competence if that was the motivation, but this is not actually needed to my understanding right?